applications/structural_application/custom_strategies/strategies/residualbased_arc_length_strategy.h

```diff
@@ -53,35 +53,33 @@ SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 #include <limits>
 #include<iostream>
 #include<iomanip>
-
+#include <cmath>
 
 /* External includes */
 
 /* Project includes */
 #include "structural_application.h"
 #include "includes/define.h"
 #include "includes/model_part.h"
+#include "containers/model.h"
```

Comment:
I think this file is included in the core. could u take a look?

Reply:
(i refer to the arc_lenght_strategy)

Reply:
Hi @RiccardoRossi , the residualbased_arc_length_strategy.h (or similar) is not yet in the core